### PR TITLE
#167728010 Setup dotenv package

### PR DESCRIPTION
**Pull request by Wokoro**
#### What does this PR do?
Setup `dotenv` package for the project

#### Description of Task to be completed?
Enable project to use enviroment variable(s) via dotenv package using *.env* file.

#### step 1
Install dotenv if not installed as a project dependency.

`npm install dotenv --save`

#### step 2
Create `.env` file in the root directory of the project.

Navigate to project root and run the command `touch .env`.

#### step 3
Add environment variables on separate lines in the `.env` file from step 2.

Example:

`.env`

    NODE_ENV=develop
    DB_HOST=localhost
    DB_USER=sample_user_name
    DB_PASS=sample_database_password

#### step 4
Require and configure dotenv and access environment variables.

Example:

`db.config.js`

    require('dotenv').config() // require and configure dotenv package.

    const db = require('db')

    db.connect({
      host: process.env.DB_HOST, // example usage of DB_HOST environment variable from .env file.
      username: process.env.DB_USER,
      password: process.env.DB_PASS
    })

#### Any background context you want to provide?
Ensure dotenv is installed before trying to use the package.

#### What are the relevant pivotal tracker stories?
[#167728010](https://www.pivotaltracker.com/story/show/167728010)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,47 @@ Barefoot Nomad - Making company travel and accomodation easy and convinient.
...
+
+    db.connect({
+      host: process.env.DB_HOST, // example usage of DB_HOST environment variable from .env file.
+      username: process.env.DB_USER,
+      password: process.env.DB_PASS
+    })
+
+
+
+For more information visit [dotenv](https://https://www.npmjs.com/package/dotenv).
```
Comment (biolabalo) [inline README.md]: remove this link (Line 49)

Comment (biolabalo) [reply README.md]: Also,  it's good practice to create an env.sample file that also mimics the env file.

Comment (Wokoro) [reply README.md]: Ok, noted.

Comment (Wokoro) [reply README.md]: I have implemented your feedback @biolabalo.

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,4 @@
+TEST_DB_URL=postgres://postgres:postgres@localhost:5432/way_farer_test
+DEV_DB_URL=postgres://postgres:postgres@localhost:5432/way_farer_dev
+PRO_DB_URL=postgres://elykwhhg:ph_oSeTukwNNbr_sizHBOcfpry2B8kFX@raja.db.elephantsql.com:5432/elykwhhg
+PRI_KEY=JKDL;AJKl;dajkle;JKLjkdla;JKLD;
```
Comment (abejide001) [inline .env.sample]: please add a new line here

**Review by abejide001 — CHANGES_REQUESTED** [review]
I left a comment

File: .env.sample (hunk 2 above)
Comment (Wokoro) [reply .env.sample]: Ok.

Comment (Wokoro) [reply .env.sample]: It has been resolved.

**Review by abejide001 — APPROVED** [review]
LGTM

**Review by Abobos — APPROVED** [review]

**Review by dieudonneAwa — APPROVED** [review]

**Pull request closed without merging**